x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.container.tsx

```diff
@@ -45,47 +40,43 @@ export const EditPolicy: React.FunctionComponent<RouteComponentProps<RouterProps
 
   if (isLoading) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
-        <EuiEmptyPrompt
-          title={<EuiLoadingSpinner size="xl" />}
-          body={
-            <FormattedMessage
-              id="xpack.indexLifecycleMgmt.editPolicy.policiesLoading"
-              defaultMessage="Loading policies..."
-            />
-          }
-        />
-      </EuiPageContent>
+      <EuiPageTemplate.EmptyPrompt
+        title={<EuiLoadingSpinner size="xl" />}
+        body={
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.policiesLoading"
+            defaultMessage="Loading policies..."
+          />
+        }
+      />
     );
   }
   if (error || !policies) {
     const { statusCode, message } = error ? error : { statusCode: '', message: '' };
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
-        <EuiEmptyPrompt
-          title={
-            <h2>
-              <FormattedMessage
-                id="xpack.indexLifecycleMgmt.editPolicy.lifecyclePoliciesLoadingFailedTitle"
-                defaultMessage="Unable to load existing lifecycle policies"
-              />
-            </h2>
-          }
-          body={
-            <p>
-              {message} ({statusCode})
-            </p>
-          }
-          actions={
-            <EuiButton onClick={resendRequest} iconType="refresh" color="danger">
-              <FormattedMessage
-                id="xpack.indexLifecycleMgmt.editPolicy.lifecyclePoliciesReloadButton"
-                defaultMessage="Try again"
-              />
-            </EuiButton>
-          }
-        />
-      </EuiPageContent>
+      <EuiPageTemplate.EmptyPrompt
```

Comment:
Please add `color="danger"` prop

```suggestion
      <EuiPageTemplate.EmptyPrompt color="danger"
```

Current
<img width="607" alt="Screenshot 2023-08-16 at 3 29 22 PM" src="https://github.com/elastic/kibana/assets/5226211/5b999723-76aa-4d30-96ca-0d7ff13b115c">

Expected
<img width="626" alt="Screenshot 2023-08-16 at 3 30 55 PM" src="https://github.com/elastic/kibana/assets/5226211/f8574c6c-a585-46f1-a1da-3fee18bbfa91">